src/diffusers/models/resnet.py

```diff
@@ -161,7 +160,7 @@ def _upsample_2d(self, x, weight=None, kernel=None, factor=2, gain=1):
 
             # Transpose weights.
             weight = torch.reshape(weight, (num_groups, -1, inC, convH, convW))
-            weight = weight[..., ::-1, ::-1].permute(0, 2, 1, 3, 4)
+            weight = torch.flip(weight, dims=[3, 4]).permute(0, 2, 1, 3, 4)
```

Comment:
Without the fix, we got
```bash
ValueError: step must be greater than zero
```
The syntax `::-1` is for TF tensors, not for torch tensors.